srcpkgs/cgal/template

```diff
@@ -1,26 +1,26 @@
 # Template file for 'cgal'
 pkgname=cgal
-version=4.14
-revision=2
+version=5.2.1
+revision=1
 wrksrc="CGAL-${version}"
 build_style=cmake
+configure_args="-DCGAL_HEADER_ONLY=ON -DBUILD_TESTING=ON"
 build_helper="qemu"
-makedepends="boost-devel eigen mpfr-devel"
+makedepends="boost-devel eigen mpfr-devel zlib-devel"
 short_desc="Computational Geometry Algorithms Library"
-maintainer="Pierre Allegraud <pierre.allegraud@crans.org>"
+maintainer="Karl Nilsson <karl.robert.nilsson@gmail.com>"
 license="GPL-3.0-or-later, LGPL-3.0-or-later"
 homepage="https://www.cgal.org"
-distfiles="https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-${version}/CGAL-${version}.tar.xz"
-checksum=59464b1eaee892f2223ba570a7642892c999e29524ab102a6efd7c29c94a29f7
+distfiles="https://github.com/CGAL/cgal/releases/download/v${version}/CGAL-${version}-library.tar.xz"
+checksum=390aa87c4f21609c19397c4b14abb5ccad3edd1e33933a0089b266f67ce7b111
+
+do_check() {
+    cd build
+    CTEST_OUTPUT_ON_FAILURE=TRUE ctest
+}
 
 cgal-devel_package() {
-	short_desc+=" - development files"
-	depends="${makedepends} ${sourcepkg}>=${version}_${revision}"
-	pkg_install() {
-		vmove usr/bin/cgal_create_cmake_script
-		vmove usr/share/man/man1/cgal_create_cmake_script.1
-		vmove usr/include
-		vmove "usr/lib/*.so"
-		vmove usr/lib/cmake
-	}
+    build_style=meta
+	depends="${sourcepkg}>=${version}_${revision} boost-devel eigen gmp-devel mpfr-devel zlib-devel"
```

Comment:
I think these depends should be moved to the main package? Actually, we can simply remove the subpackage and add `replaces="cgal-devel>=0"` to the template; it will be pulled in automatically, since the previous `cgal-devel` already depended on it.